README.md

```diff
@@ -114,4 +114,4 @@ npx source-map-explorer 'packages/code-studio/build/static/js/*.js'
 
 ## Updating Snapshots
 
-Snapshots are used by end-to-end tests to visually verify the output. Sometimes changes are made requiring snapshots to be updated. Since snapshots are platform dependent, you may need to use a docker image to [update snapshots for CI](https://playwright.dev/docs/test-snapshots). Run `npm run e2e:update-ci-snapshots` to mount the current directory into a docker image and re-run the tests from there. **Note:** You must have [Docker installed](https://docs.docker.com/get-docker/), and `deephaven-core` must already be running on port 10000 on your local machine.
+Snapshots are used by end-to-end tests to visually verify the output. Sometimes changes are made requiring snapshots to be updated. Run snapshots locally to update first, by running `npm run e2e:update-snapshots`. Once you are satisfied with the snapshots and everything is passing locally, you may need to use a docker image to [update snapshots for CI](https://playwright.dev/docs/test-snapshots) (since snapshots are platform dependent). Run `npm run e2e:update-ci-snapshots` to mount the current directory into a docker image and re-run the tests from there. **Note:** You must have [Docker installed](https://docs.docker.com/get-docker/), and `deephaven-core` must already be running on port 10000 on your local machine.
```

Comment:
Maybe break this into 2 paragraphs. 2nd starting w/ the "Once you are satisfied".

I would change the wording to just say "you need to run npm run e2e:update-ci-snapshots" and then the short explanation why. But instead of "may need" just "need" since it's a relatively high chance (at least since most of the devs have Macs) and probably just good practice.